conf/config.yaml.sample

```diff
@@ -69,7 +69,7 @@ cortx:
       - tcp://hax.seagate.com:28000
   csm:
     auth_admin: admin
-    csmauth_secret: auth-secret
+    csmauth_secret: csmauth-secret
```

Comment:
This file is old, please use latest file from Tanuja..

Reply:
Update key as per latest config file.

`auth_secret: csmauth-secret`

Reply:
Nope its wrong

Reply:
Change S3-Auth secret as follows:
`auth_secret: s3-auth-secret`

Reply:
Updated secrets keys to:
csm-auth-secret
csm-mgmt-secret
s3-auth-secret